src/Subscription/Subscription/help/Update-AzSubscription.md

```diff
@@ -0,0 +1,153 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Subscription.dll-Help.xml
+Module Name: Az.Subscription
+online version:
+schema: 2.0.0
+---
+
+# Update-AzSubscription
+
+## SYNOPSIS
```

Comment:
please complete synopsis/description and examples

Reply:
Added the details